README.md

```diff
@@ -1 +1,226 @@
 # leto-modelizer-plugin-template
+
+This project is a template to help you write your own plugin.
+
+You can find in all js objects in `src` an explanation of what you have to do.
+
+## Update your package.json
+
+In `package.json` and in this project we provide you many thing to start your plugin development:
+- Setup jest for unit test.
+- Setup eslint with AirBnB syntax.
+- Setup the build command.
+
+Feel free to remove or use previous thing.
+
+But before starting develop your plugin you have to modify some properties of your `package.json` :
+
+- `name`: put your plugin name here.
+- `version`: we set the template version here, but you have to set your plugin version instead.
+- `description`: explain what your plugin does.
+- `repository`: update with your repository values.
+- `keywords`: Your keywords that describe your plugin.
+- `author`: put the name and email of your maintainer for your plugin.
+- `bugs` / `homepage`: update with your repository url.
+- `main` / `exports`:  correspond to your build file that would be used by other
+
+Don't forget to regenerate the `package-lock.json` with this command `npm i --package-lock-only`.
+
+Don't forget to modify the `webpack.config.js` to set your wanted export file name.
+
+We provide a `changelog.json` to explain all change in the template, you can remove it and have your own changelog file.
+
+## Name your plugin
+
+By default, in this template we use "MyPlugin" as your plugin name.
+
+So once your plugin's name is choose, you have to make these modifications:
+
+* Rename these files and their content:
+  * `src/draw/FruitDrawer.js`
+  * `src/metadata/FruitMetadata.js`
+  * `src/parser/FruitParser.js`
+  * `src/render/FruitRenderer.js`
+* Change the content of `src/index.js`
+
+Example: You want to name your plugin `Fruit`
+
+`src/draw/FruitDrawer.js` becomes `src/draw/FruitDrawer.js`
```

Comment:
> `src/draw/FruitDrawer.js` becomes `src/draw/FruitDrawer.js`

I assume the first one should be "BaseDrawer.js" ?

Reply:
yes :)

Reply:
done